### PR TITLE
Changes of repository.

### PR DESCRIPTION
**Pull request by ecdssantos**
# This is a test of PR by a fork.

How does it work?

### CODE REVIEW

**Comment by ecdssantos** [comment]
What exactly must be done?

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# test
```
Comment (evertoncss) [inline README.md]: The title must be in upper case.

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+# Changes made in a fork.
```
Comment (evertoncss) [inline README.md]: Explain more about what is done.

File: readme.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+This is only to have another change that must be commited.
```
Comment (evertoncss) [inline readme.txt]: This text can be more elaborated.

**Comment by ecdssantos** [comment]
All the requests where answered in the new commit.

**Pull request merged**